lib/presentation/components/field_picker.dart

```diff
@@ -1,38 +1,73 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:healthy_routine_mobile/resources/styles/colors.dart';
+import 'package:healthy_routine_mobile/healthy_routine.dart';
+
 
 class FieldPicker extends StatelessWidget {
   final IconData icon;
   final String label;
   final String value;
+  final FieldPickerTypes type;
+  final Function(TimeOfDay) onTap;
```

Comment:
This block won't be called on tap. I'd change it to onChange or something, what do you think?

Reply:
Yep
Done